css/base/base.css

```diff
@@ -50,7 +50,6 @@ button {
   justify-content: center;
   font-family: inherit;
   background-color: transparent;
-  outline: none;
   cursor: pointer;
 }
 
```

Comment:
- [ ] Hey, Great job on your accessibility, however there are some issues you still need to address. I'd pointing them out below, Please kindly look them up in your file and fix them. I just used this inline comment to be able make the review..

- [ ] Hey, Please kindly add the label tag to your form input and textarea.  A form should have the label tag as the label tag point or specify a label for an <input>, <select>, <textarea> element of a form to improve accessibility for screen users. There are many ways to add the label tag. use this for yours

`<label for="username"><input type="text" name="username" id="username" maxlength="30" placeholder="username" required></label>`

Reply:
- [ ] please, kindly fix this color contrast issue.
- Use this [chrome extetnsion](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en) to analyze and fix all your color contrast issue.
- Your `LET'S CONNECT`.
- Your contact section background color.
- Your list items e.g: `html css javascript`.

Reply:
## Hi @zieeco,

Thank you for your review and feedback! However, I got a question about this required change:

> * [ ]  Hey, Please kindly add the label tag to your form input and textarea.  A form should have the label tag as the label tag point or specify a label for an , , <textarea> element of a form to improve accessibility for screen users. There are many ways to add the label tag. use this for yours
>   
>   
>   <label for="username"><input type="text" name="username" id="username" maxlength="30" placeholder="username" required></label>

If I add a ``label`` tag in the way that was suggested, I will change the original design proposed by the [Figma Template 1](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1). I wouldn't like to do that because one of the core project requirements is to stick to this template design as much as possible. 

So, I thought of another solution that will achieve accessibility without changing the original design, which is to add a ``aria-label`` attribute to the form inputs and textarea. This technique is used for example in the input of the login form of Google (which doesn't have a ``label`` tag associated with it but an ``aria-label`` attribute instead):

![google-login-form](https://user-images.githubusercontent.com/64566209/166605641-856a2920-ef5b-44e1-81fd-401aa78c55d9.png)

This ``aria-label`` will provide a label for the input and textarea objects that can be read by assistive technology (e.g. screen readers) without changing my portfolio design. **So, what do you think?**